modules/ossm-identifying-revision-name.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+// install/ossm-sidecar-injection
+
+:_mod-docs-content-type: Concept
+[id="ossm-identifying-revision-name_{context}"]
+= Identifying the revision name
+
+The label required to enable sidecar injection is determined by the specific control plane instance, known as a revision. Each revision is managed by an `IstioRevision` resource, which is automatically created and managed by the `{istio}` resource, so manual creation or modification of `IstioRevision` resources is generally unnecessary.
+
+The naming of an `IstioRevision` depends on the `spec.updateStrategy.type` setting in the `{istio}` resource. If set to `InPlace`, the revision shares the `{istio}` resource name. If set to `RevisionBased`, the revision name follows the format `<Istio resource name>-v<version>`. Typically, each `{istio}` resource corresponds to a single `IstioRevision`. However, during a revision-based upgrade, multiple `IstioRevision` resources may exist, each representing a distinct control plane instance.
+
+To see available revision names, use the following command:
+
+[source,terminal]
+----
+$ oc get istiorevisions
+----
+
+You should see output similar to the following example:
+
+.Example output
+[source,terminal]
+----
+NAME              READY   STATUS    IN USE   VERSION   AGE
+my-mesh-v1-23-0   True    Healthy   False    v1.23.0   114s
+----
+
+[id="ossm-identifying-revision-name-default_{context}"]
+== Enabling sidecar injection with default revision
+
+When the service mesh's `IstioRevision` name is `default`, it's possible to use the following labels on a namespace or a pod to enable sidecar injection:
+
+[options="header"]
+|===
+|Resource |Label |Enabled value |Disabled value
+
+|Namespace |`istio-injection` |`enabled` |`disabled`
+
+|Pod |`sidecar.istio.io/inject` |`true` |`false`
+|===
+
+[NOTE]
+====
+You can also enable injection by setting the label `istio.io/rev: default` on the namespace or the pod.
+====
+
+[id="ossm-identifying-revision-name-other_{context}"]
+== Enabling sidecar injection with other revisions
+
+When the `IstioRevision` name is not `default`, then the specific `IstioRevision` name must be used with the `istio.io/rev` label to map the pod to the desired control plane while enabling sidecar injection. To enable injection, set the `istio.io/rev: default` label on either the namespace or the pod. Setting the label on both is not required.
```

Comment:
Avoid passive tense. So maybe rewrite as: "...you must use the specific `IstioRevision` name..."